FEM/pcs_dm.cpp

```diff
@@ -772,19 +772,21 @@ double CRFProcessDeformation::Execute(int loop_process_number)
 
 				Error = Norm / InitialNorm;
 				ErrorU = NormU / InitialNormU0;
-				if (Norm < Tolerance_global_Newton && Error > Norm)
```

Comment:
@wenqing @norihiro-w  I've checked the benchmarks. There are some M/HM (mainly UJG, WX) benchmarks failing due to tolerance settings. The solutions are "correct", it's only a matter of adjusting convergence settings and replacing reference files. Nothing actually "fails". I suggest I take care of the benchmarks after this PR has been merged.